backend/controllers/postController.js

```diff
@@ -102,42 +145,92 @@ getPosts = async function(req,res){
 						[Op.or] : [
 							{'$project_collaborators.user_id$' : {[Op.eq]: user_id}},
 							{'$project.privacy$' : {[Op.eq]: 1}}
-					]}
+						]}
+					]
+					},
+					include : [
+						{
```

Comment:
for both cases of if else, include is the same as far as i see, instead of copy paste'in the same code, you might keep that in a variable and lter reference that variable, this really would increase the readability and simplify later maintaining.

Reply:
yeah, its better, changed it.